doc/source/contributing/index.rst

```diff
@@ -124,22 +124,25 @@ To run all the tests (except the virtual instruments)
 .. code-block:: sh
 
     cd scikit-rf
-    pytest
+    python -m venv .venv
```

Comment:
OK. 

I would just suggest to add a bit of comments before or here about what all these commands and `venv` do, for the ones not expert in modern Python tools (most of users certainly) (since before it was only `pytest`).

Reply:
yes before it was only pytest, but in fact this does not work most of the time, because usually pytest is not installed system wide.